src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,72 @@
-import React from 'react';
+import React, { useState, useCallback } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+
+import { MovieCard } from '../MovieCard';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  addMovie: (movie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
  const [isError, setIsError] = useState(false);
```
![image](https://user-images.githubusercontent.com/96409563/225120377-b79da9c9-237a-4ede-a496-7992dd0332a5.png)